modules/ROOT/pages/7.6.0-release-notes.adoc

```diff
@@ -153,10 +153,12 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 {productname} {release-version} also includes the following bug fix<es>:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Tooltips of toolbar groups not showing
```

Comment:
Please update to match the `.changes/unreleased/tinymce-TINY-11391-2024-10-28.yaml` entry so it aligns with changelog when its generated.
```suggestion
=== Tooltip would not show for group toolbar button.
```